include/boost/geometry/algorithms/detail/relate/point_point.hpp

```diff
@@ -67,20 +67,25 @@ std::pair<bool, bool> point_multipoint_check(Point const& point,
     // point_in_geometry could be used here but why iterate over MultiPoint twice?
     // we must search for a point in the exterior because all points in MultiPoint can be equal
 
-    typedef typename boost::range_iterator<MultiPoint const>::type iterator;
-    iterator it = boost::begin(multi_point);
-    iterator last = boost::end(multi_point);
-    for ( ; it != last ; ++it )
+    auto it = boost::begin(multi_point);
+    auto last = boost::end(multi_point);
+    for ( ; it != last ; ++it)
```

Comment:
These iterators could probably be put inside the loop. At least `it` because with `last`/`end` there could be a debate about performance.

Reply:
:heavy_check_mark: fixed and renamed `last` to `end` as done elsewhere